tests/Oryx.BuildImage.Tests/Node/NodeJSSampleAppsTestBase.cs

```diff
@@ -24,5 +28,43 @@ public class NodeJSSampleAppsTestBase : SampleAppsTestBase
             }))
         {
         }
+
+        [Fact]
+        public void GeneratesScript_AndLoggerFormatCheck()
+        {
+            // Arrange
+            var volume = CreateWebFrontEndVolume();
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/" + SampleAppName + "-output";
+            var script = new ShellScriptBuilder()
+                .AddDefaultTestEnvironmentVariables()
+                .AddCommand($"echo RandomText >> {appDir}/Program.cs") // triggers a failure
+                .AddBuildCommand(
+                $"{appDir} -o {appOutputDir}")
+                .ToString();
+            // Regex will match:
+            // "yyyy-mm-dd hh:mm:ss"|ERROR|Micro
```

Comment:
Will the regex actually match for `"yyyy-mm-dd hh:mm:ss"|ERROR|Micro` per comment above?

Reply:
`Micro` is missing. So, it should not match.

Reply:
Also, it should be a warning.